packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -138,7 +138,7 @@ function RevisionsButtons( {
 	return (
 		<ol
 			className="edit-site-global-styles-screen-revisions__revisions-list"
-			aria-label={ __( 'Global styles revisions' ) }
+			aria-label={ __( 'Global styles revisions list' ) }
```

Comment:
Changed because there's already a label `Global styles revisions` on the editor canvas container when revisions is active.